loopchain/peer/consensus_siever.py

```diff
@@ -96,36 +96,38 @@ def __build_candidate_block(self, block_builder, next_leader):
         vote = await self._wait_for_voting(block)
         if not vote:
             raise NotEnoughVotes
+        elif not vote.get_result():
+            raise InvalidBlock
 
         self._blockchain.add_block(block, confirm_info=vote.votes)
         self._block_manager.candidate_blocks.remove_block(block.header.hash)
         self._blockchain.last_unconfirmed_block = None
 
+    def __get_complaint_votes(self):
+        if self._block_manager.epoch.round > 0:
+            return self._block_manager.epoch.complain_votes[self._block_manager.epoch.round - 1]
+        return None
+
     async def consensus(self):
+        util.logger.debug(f"-------------------consensus-------------------")
```

Comment:
develop branch has this comment already. see develop when rebase with it.

Reply:
👌